protos/processor.proto

```diff
@@ -50,6 +50,32 @@ message TpRegisterResponse {
     Status status = 1;
 }
 
+// The unregistration request from the transaction processor to the validator/executor
+message TpUnRegisterRequest {
```

Comment:
Another possible approach would be to send less information in this request, then find the processors based on identity instead of (family, version, encoding).  Presumably that approach would be more of an unregister-all message.